src/main/resources/messages/LibertyBundles.properties

```diff
@@ -104,16 +104,16 @@ gradle.test.report.does.not.exist=Gradle Test Report Does Not Exist
 test.summary=Test Summary
 
 # View unit test report action
-view.unit.test.report=view unit test report
+view.unit.test.report=View unit test report
 unit.test.report.does.not.exist=Unit Test Report Does Not Exist
 
 # Run & Debug Configuration
 liberty.run.config.title=Liberty Run Configuration
 liberty.debug.port.unresolved=Unable to {0} in debug mode: could not resolve debug port for {1}.
-attaching.debugger=Attaching Debugger to Liberty Server JVM...
+attaching.debugger=Attaching Debugger
 cannot.connect.debug.port=Cannot connect debugger to port {0}.
-run.config.liberty.project=Liberty Project
-run.config.liberty.project.tool.tip=Select the build file for your Liberty project
+run.config.liberty.project=Project
+run.config.liberty.project.tool.tip=Select the build file for your Project
```

Comment:
What was the reason for taking Liberty out of these messages?

Reply:
Will reinsert 'Liberty tool window'
I took the VS Code wording too literally. I can reinsert 'Liberty'